### PR TITLE
[website] ui icons

### PR DESCRIPTION
**Pull request by stefanorigano**
Add new SVG icons for the website editor UI.
Task: 2157252

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/website/views/snippets/snippets.xml [hunk 1]
```diff
@@ -686,12 +686,22 @@
...
     <!--  Vertical Alignment -->
     <div id="row_valign_snippet_option" data-selector=".s_text_image, .s_image_text, .s_three_columns" data-target=".row">
-        <we-button-group string="Vert. Alignment" title="Vertical Alignment">
-            <we-button title="Top" data-select-class="align-items-start"><i class="fa fa-fw fa-long-arrow-up"/></we-button>
-            <we-button title="Middle" data-select-class="align-items-center"><i class="fa fa-fw fa-circle"/></we-button>
-            <we-button title="Bottom" data-select-class="align-items-end"><i class="fa fa-fw fa-long-arrow-down"/></we-button>
-            <we-button title="Equal height" data-select-class="align-items-stretch"><i class="fa fa-fw fa-arrows-v"/></we-button>
-        </we-button-group>
+        <we-row>
+            <we-button-group string="Vert. Alignment" title="Vertical Alignment">
```
Comment (qsm-odoo) [inline addons/website/views/snippets/snippets.xml:690]: @stefanorigano I don't know why you wrapped in a we-row... I removed that ^^

**Comment by qsm-odoo** [comment]
@robodoo rebase-ff r+

File: addons/website/views/snippets/snippets.xml (hunk 1 above)
Comment (stefanorigano) [reply addons/website/views/snippets/snippets.xml]: @qsm-odoo , sorry, leftover , you can remove it of course :)

Thanks

**Comment by qsm-odoo** [comment]
@robodoo r+

**Comment by qsm-odoo** [comment]
@robodoo r+

**Pull request closed without merging**